webui/react/src/utils/task.ts

```diff
@@ -30,3 +31,6 @@ export function generateTasks(): RecentTask[] {
     return props;
   });
 }
+
+// FIXME should check for type specific conditions eg tensorboard is terminated
+export const canBeOpened = (task: Task): boolean => !!task.url;
```

Comment:
question: should this be addressed in this PR rather than later or is the effort significant?

Reply:
no, I think it'll be pretty contained.  it would apply to the dashboard as well.

Reply:
done